lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/Sequential.hs

```diff
@@ -152,7 +152,7 @@ instance ToText (AddressPoolGap) where
 
 instance Bounded AddressPoolGap where
     minBound = AddressPoolGap 10
-    maxBound = AddressPoolGap 100
+    maxBound = AddressPoolGap 100000
```

Comment:
:+1:

Reply:
someone was having 130k addresses, why not to change it to 150k ?

Reply:
They only had 40k _unused_ addresses if I recall correctly, remember that the gap is about the number of **unused consecutive** addresses. So I am starting with 100_000 (which seems to take 8 hours to restore on mainnet :upside_down_face:) which should be sufficient ... I hope.